src/scripts/num.js

```diff
@@ -135,7 +135,7 @@ export function atoms(number = 0) {
 export function uatoms(number = 0) {
   return BigNumber(number)
     .times(1e6)
-    .toFixed(0)
+    //.toFixed(0) // removing this will solve the issue with fee amount if extension

```

Comment:
this function converts view values of tokens into internal values. i.e. converts 1.5atoms into 1500000 uatoms. uatoms never have decimals.

If adding decimals to uatoms. I guess the problem lies in a different location.

Reply:
Ok, then we need to rewrite fee defining function in signRequest